restock.lic

```diff
@@ -0,0 +1,107 @@
+#   Documentation: https://elanthipedia.play.net/Lich_script_repository#restock
+
+custom_require.call(%w[common common-items common-money])
+
+class Restock
+  include DRC
+  include DRCM
+  include DRCI
+
+  def initialize
+    setup
+
+    items = parse_restockable_items
+    items_to_restock = []
+    coin_needed = 0
+
+    items.each do |item|
+      remaining = if item['stackable']
+                    count_stackable_item(item['name'])
+                  else
+                    count_nonstackable_item(item['name'])
+                  end
+
+      next unless remaining < item['quantity']
```

Comment:
Is this the same as `next if remaining >= item['quantity']`?

Reply:
yes. pretty sure rubocop made that change.